ts/features/idpay/details/utils/hooks.tsx

```diff
@@ -82,3 +92,53 @@ export const useInitiativeTimelineFetcher = (
     lastUpdate
   } as const;
 };
+
+export const useIdpayDiscountDetailsBottomSheet = (initiativeId: string) => {
```

Comment:
```suggestion
export const useIdPayDiscountDetailsBottomSheet = (initiativeId: string) => {
```

Another thing: imo this `hooks.ts` file is growing pretty fast. What do you think about a `hooks` folder where to place all hooks inside?
In this case we would have:
- `ts/features/idpay/details/hooks/useIdPayDiscountDetailsBottomSheet.tsx`

Reply:
sure, I don't see any reason why not 
That said, can I mark your previous comment as resolved? to me it seems like this one resolves the last one, LMK👍

Reply:
addressed in [064d125](https://github.com/pagopa/io-app/pull/5146/commits/064d12534a39bcb8dcf1130b00d8002d23f01c45)